modules/minimum-required-permissions-ipi-azure.adoc

```diff
@@ -4,18 +4,9 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="minimum-required-permissions-ipi-azure_{context}"]
-= Required Azure permissions for installer-provisioned infrastructure
+= Required {azure-short} permissions for installer-provisioned infrastructure
 
-The installation program requires access to an Azure service principal or managed identity with the necessary permissions to deploy the cluster and to maintain its daily operation. These permissions must be granted to the Azure subscription that is associated with the identity.
-
-The following options are available to you:
-
-* You can assign the identity the `Contributor` and `User Access Administrator` roles. Assigning these roles is the quickest way to grant all of the required permissions.
-+
-For more information about assigning roles, see the Azure documentation for link:https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal[managing access to Azure resources using the Azure portal].
-* If your organization's security policies require a more restrictive set of permissions, you can create a link:https://learn.microsoft.com/en-us/azure/role-based-access-control/custom-roles[custom role] with the necessary permissions.
-
-The following permissions are required for creating an {product-title} cluster on Microsoft Azure.
+The following permissions are required for creating an {product-title} cluster on {azure-first}.
```

Comment:
I don't think original line9 should be removed. It describes that which role can be assigned to identity on scope of subscription. 
1. system role `Contributor` and `User Access Administrator`
2.  or creating customer role with following permissions
so suggest to add it back, just remove the duplicated sentence.
```
These permissions must be granted to the Azure subscription that is associated with the identity.
Comment

The following options are available to you:

* You can assign the identity the `Contributor` and `User Access Administrator` roles. Assigning these roles is the quickest way to grant all of the required permissions.
+
For more information about assigning roles, see the Azure documentation for link:https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal[managing access to Azure resources using the Azure portal].
* If your organization's security policies require a more restrictive set of permissions, you can create a link:https://learn.microsoft.com/en-us/azure/role-based-access-control/custom-roles[custom role] with the necessary permissions.

The following permissions are required for creating an {product-title} cluster on Microsoft Azure.
```

Reply:
Or moved into section `Required Azure roles`, https://79423--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-account.html#installation-azure-permissions_installing-azure-account

Reply:
It's already present in https://79423--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-account.html#installation-azure-permissions_installing-azure-account

Reply:
> It's already present in https://79423--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-account.html#installation-azure-permissions_installing-azure-account

They are different. The permissions mentioned in your link is used to create Service principal or Managed identities.

And here is required roles to be assigned on created Service principal or Managed identities, then use them to create cluster.